backend/src/config.rs

```diff
@@ -0,0 +1,39 @@
+use serde::{Deserialize, Serialize};
+
+use std::io::Write;
+use std::{fs, path};
+
+#[derive(Clone, Serialize, Deserialize)]
+pub struct Config {
+    // Address with port, e.g. "localhost:8088"
+    pub address: String,
+}
+
+impl Config {
+    // Create default config file
+    pub fn new() -> Self {
+        Config {
+            address: "localhost:8088".to_string(),
+        }
+    }
+}
+
+lazy_static! {
+    pub static ref CONFIG: Config = get_initial_config();
```

Comment:
i have a doubt about initializing config using lazy_static

maybe we would not need config anywhere except main function...
I'm also telling this because there should be panic when there is an error while config reading and u did it just right for me

also. the only moment when we should panic is startup. i see that now `get_initial_config` has it's first call in main function, but it makes behavior of whole code less obvious

Reply:
>maybe we would not need config anywhere except main function...

We would.
I introduced config also for the purpose of storing JWT parameters, which will be used in authentication module.

>i see that now get_initial_config has it's first call in main function
It has? It shouldn't, thought. Static initialization should occur before any code from main runs. Also static data is a common concept in statically compiled languages, so it's safe to assume that any static initialization will have finished (or panicked) by the time the `main` starts.

Reply:
It's not the case for __lazy__ static, though. It indeed gets initialized on the first dereference, which is indeed not that obvious. Perhaps we indeed should add explicit static initialization in the main function, but personally I see this as an overkill.

Reply:
Oh, i didn't got it
I thought you run lazy function when code first call it and next calls just get value that was calculated on that very first run
And you use it in bind function, so first call is there